doc/plugins/org.polarsys.capella.core.ui.intro/src/org/polarsys/capella/core/ui/intro/views/WelcomeView.java

```diff
@@ -0,0 +1,314 @@
+
+      gc.setFont(introTextFont);
+      gc.drawText("Getting started with Capella ", 75, 130, true);
+
+    });
+
+    viewCanvas.setBackgroundMode(SWT.INHERIT_FORCE);
+
+    Composite itemsComposite = new Composite(intermediaryComposite, SWT.FILL);
+    itemsComposite.setBackgroundMode(SWT.INHERIT_FORCE);
+    itemsComposite.setBackground(parent.getDisplay().getSystemColor(SWT.COLOR_WHITE));
+
+    GridData itemsCompositeLayoutData = new GridData();
+    itemsCompositeLayoutData.horizontalAlignment = SWT.FILL;
+    itemsCompositeLayoutData.verticalAlignment = SWT.FILL;
+    itemsComposite.setLayoutData(itemsCompositeLayoutData);
+
+    GridLayout itemsLayout = new GridLayout();
+    itemsComposite.setLayout(itemsLayout);
+    itemsLayout.numColumns = 2;
+
+    Composite leftPanel = new Composite(itemsComposite, SWT.FILL);
+    leftPanel.setBackgroundMode(SWT.INHERIT_FORCE);
+    leftPanel.setBackground(parent.getDisplay().getSystemColor(SWT.COLOR_WHITE));
+    GridLayout leftLayout = new GridLayout();
+    leftPanel.setLayout(leftLayout);
+    leftLayout.numColumns = 1;
+
+    Composite rightPanel = new Composite(itemsComposite, SWT.FILL);
+    rightPanel.setBackgroundMode(SWT.INHERIT_FORCE);
+    rightPanel.setBackground(parent.getDisplay().getSystemColor(SWT.COLOR_WHITE));
+    GridLayout rightLayout = new GridLayout();
+    rightPanel.setLayout(rightLayout);
+    rightLayout.numColumns = 1;
+
+    initializeContributions();
+    HyperlinkGroup group = new HyperlinkGroup(itemsComposite.getDisplay());
+    int numberOfContributions = contributions.size();
+    int count = 0;
+    for (WelcomePageContribution contrib : contributions) {
+
+      // Create a Wrapper composite to contain the ImageHyperLink and its Label
+      Composite parentComposite = null;
+      if (count <= numberOfContributions / 2) {
+        parentComposite = leftPanel;
+      } else {
+        parentComposite = rightPanel;
+      }
+      Composite contribComposite = new Composite(parentComposite, SWT.FILL);
+
+      FillLayout contribLayout = new FillLayout(SWT.VERTICAL);
+      // contribLayout.numColumns = 1;
+
+      GridData contribLayoutData = new GridData();
+      contribLayoutData.widthHint = 400;
+      contribComposite.setLayout(contribLayout);
+      contribComposite.setLayoutData(contribLayoutData);
+
+      // Create a ImageHyperlink
+      ImageHyperlink hyperLink = new ImageHyperlink(contribComposite, SWT.WRAP);
+      hyperLink.setText(contrib.getTitle());
+      hyperLink.setHref(contrib.getLink());
+      hyperLink.setImage(contrib.getIcon());
+      hyperLink.addHyperlinkListener(contrib.getHandler());
+
+      // Custom BOLD Font
+      Font formerFont = hyperLink.getFont();
+      FontData[] formerFontData = formerFont.getFontData();
+      formerFontData[0].setStyle(formerFontData[0].getStyle() | SWT.BOLD);
+      hyperLink.setFont(new Font(parent.getDisplay(), formerFontData[0]));
```

Comment:
The font shall be disposed at the end.
It's better to initialize it first than creating one for each links.

you can increase a bit the size. (.setHeight());
![image](https://github.com/eclipse/capella/assets/34767327/b8094c62-15a9-4d0a-a569-a5c50f4c4d16)